posts/2022/01/resuming-writing-challenge.md

```diff
@@ -0,0 +1,39 @@
+<!--
+.. title: On Resuming Writing Challenge
+.. slug: resuming-writing-challenge
+.. date: 2022-01-11 06:06:06 UTC+05:30
+.. tags: personal, writing
+.. category:
+.. link:
+.. description: 1,00,000 reward to readers who calls out that there is no new blog post every month.
+.. type: text
+-->
+
+<p align="center">
+<img height="300" width="800" src="/images/writing_challenge.jpg" />
+</p>
+
+<p align="center">
+Photo by Kaboompics on Pexels
+</p>
+
+
+In 2018, I decided to write at least one blog post per month throughout the year. Even though I tried to write posts every month, I couldn't publish anything in few months.
+
+In 2019, I went a step ahead and made a legal(?) agreement with a friend. I paid him 1,00,000 rupees and told him that he could keep the money as a reward if I failed to write a blog post every month.
+
+This agreement kept me on my toes. I didn't miss writing a single month in 2019. I stayed awake on the last days of the month to finish and publish the post before midnight.
+
+In 2020, I took up the challenge again and I was able to write at least one post every month.
+
+In 2021, I didn't take up the challenge. I wrote just three posts in the entire year.
+
+In the two years when I took the challenge, even though I wrote a few mediocre articles, I wrote a few good articles. In the other two years when I didn't take the challenge, my writing quality and quantity declined.
+
+Due to this, I decided to take up the writing challenge again this year.
+
+Instead of limiting the 1,00,000 reward to my friend, I decided to extend it to all the readers.
```

Comment:
I think you should keep this model for 2022 too. I am happy to accept 1L INR from you :)

Reply:
@darshanime  I am keeping this model for 2022 too.

Hopefully, I will write 1 blog post per month :-)